lib/api/pokedex_api_spec/model/pokemon_model.dart

```diff
@@ -9,6 +9,7 @@ class Pokemon with _$Pokemon {
   factory Pokemon({
     @Default('') @JsonKey(name: 'name') String name,
     @Default('') @JsonKey(name: 'url') String url,
+    @Default('') String img,
```

Comment:
is this coming from the structure response from the api? Why do you need this here?

also kindly rename your dart file here based on the class name you provided 🙂

Reply:
its not part of the json response but I included it to attach the img of the pokemon to its model

Reply:
yep, then see feedback here 🙂 :

https://github.com/shawn-ffufm/pokedex-async-redux/pull/17#discussion_r939956203